lib/logger/lib/logger.ex

```diff
@@ -88,19 +88,35 @@ defmodule Logger do
       #{perform_cleanup()}")` as in the example `perform_cleanup/0` won't be
       executed if the `:compile_time_purge_level` is `:info` or higher.
 
+    * `:compile_time_purge_matching` - purges *at compilation time* all calls
+      that matches the given metadata. This configuration expects a list of
+      keyword lists. Each keyword list contains a metadata key and the matching
+      value that should be purged. Remember that if you want to put purge log
```

Comment:
Maybe "Remember that if you want to purge log calls from a dependency, the dependency must be recompiled."?